test/integration/deploy_trigger_test.go

```diff
@@ -197,8 +206,19 @@ func TestTriggers_imageChange(t *testing.T) {
 		t.Fatalf("Expected deployment annotated with deploymentConfig '%s', got '%s'", e, a)
 	}
 
+	// expecting RC update from new to pending
```

Comment:
Hence my comment on IRC about possible races. I'm thinking that this is ok, because after the test creates a DeploymentConfig, I don't believe anything will be operating on it besides the DeploymentController, which changes the status from new to pending. But it might still make more sense to write this test in a more guarded manner. I'd like to wait until @ironcladlou is back so he can comment as well.